tgui/packages/tgui/components/DepartmentPane.tsx

```diff
@@ -0,0 +1,73 @@
+import { Color } from 'common/color';
+import { SFC } from 'inferno';
+import { Box } from './Box';
+import { Stack } from './Stack';
+
+// The cost of flexibility and prettiness.
+export const DepartmentEntry: SFC<{
+  style?;
+  titleStyle?;
+  textStyle?;
+  title?;
+  titleContents?;
+  titleSubtext?;
+}> = (props) => {
+  return (
+    <Box ClassName="Section" style={props.style}>
+      <Box class="Section__title" style={props.titleStyle} minHeight="3.4rem">
+        <Box class="Section__titleText" style={props.textStyle}>
+          {props.title}
+          {props.titleContents}
+        </Box>
+        {props.titleSubtext}
+        <br style={{ 'clear': 'both' }} />
+      </Box>
+      <Box class="Section__rest">
+        <Box class="Section__content">{props.children}</Box>
+      </Box>
+    </Box>
+  );
+};
+
+export const DepartmentPane: SFC<{
+  act;
+  departments: object[];
```

Comment:
Create a type for the department instead of just using `object`, here and with job too

Reply:
Passing `act` is also unnecessary, you can get it from useBackend.

Reply:
`useBackend` can't be done here without passing a type, as far as I know. This pane is not specific to a single page, and will be reused in the prefs menu at the very least, so you're aware.

Reply:
Oh wait, I feel stupid. Nevermind, I never use it here in the first place.